src/plone/app/mosaic/transform.py

```diff
@@ -152,10 +152,18 @@ def transformIterable(self, result, encoding):
             ])
 
         # Get contentLayout body class
-        if 'template-layout' in body_classes:
+        if (
+            'template-layout' in body_classes or
```

Comment:
I see... that differs from plone.app.layout, where template-layout_view is added.
might nothing to care about, though.